cinch/site.yml

```diff
@@ -31,6 +31,17 @@
   roles:
     - repositories
 
+- name: enable additional package streams on RHEL 8
+  become: true
+  hosts: repositories
+  tasks:
+    - name: enable libselinux-python stream
+      shell: dnf -y module enable libselinux-python
+      when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "8"
+    - name: enable Java stream
+      shell: dnf -y module enable javapackages-runtime
```

Comment:
Is this needed for running Jenkins?
Because there is `jslave_extra_rpms` for [customizing jenkins slave deployment](cinch/roles/jenkins_slave/defaults/main.yml) and similar for Jenkins master `jmaster_extra_rpms`